src/server_manager/install_scripts/do_install_server.sh

```diff
@@ -34,11 +36,11 @@ export SHADOWBOX_DIR="${SHADOWBOX_DIR:-${HOME:-/root}/shadowbox}"
 mkdir -p $SHADOWBOX_DIR
 
 # Save output for debugging
-exec 2>&1 >$SHADOWBOX_DIR/install-shadowbox-output
+exec >$SHADOWBOX_DIR/install-shadowbox-output 2>&1
 
 # Initialize sentry log file.
 export SENTRY_LOG_FILE="$SHADOWBOX_DIR/sentry-log-file.txt"
-> $SENTRY_LOG_FILE
+true > $SENTRY_LOG_FILE
```

Comment:
I don't understand this one. Why do we need true?

Reply:
It took me a long time to understand this code.  I think what this code was trying to do is to make sure that `$SENTRY_LOG_FILE` exists, and is empty, by sending a null input to it in truncating mode.  (Similarly for `$ACCESS_LOG` later.)  What it was actually doing was redirecting all subsequent STDOUT output from *this script* to that file (instead of `install-shadowbox-output`).

`true` is a command that prints nothing and returns 0.  This construction is shellcheck's recommended solution.